src/connection/Connection.ts

```diff
@@ -106,7 +111,8 @@ export class Connection implements IConnection {
       else valueToSend = value;
     });
 
-    if (valueToSend !== {}) messages.push({ topic, propertyName: current, value: valueToSend });
+    // the condition `if (valueToSend !== {}) ` has been removed bc it always evaluates to true
```

Comment:
**thought**: maybe here is worth understanding if is better to remove the `if` or leaving the `if` fixing the condition

Reply:
Ok, it makes sense, I'm going to see if I can understand the original intention